### PR TITLE
test: repro enums in tsx undefined

### PR DESCRIPTION
**Pull request by rwoll**
The enums imported from the .tsx file are ending up as "undefined";
from the .ts file they are defined.

Repro for #13265.

This likely has to do with our component testing/transform; before fixing, I want to
understand expected behavior.

### CODE REVIEW

File: tests/playwright-test/babel.spec.ts [hunk 1]
```diff
@@ -36,3 +36,43 @@ test('should succeed', async ({ runInlineTest }) => {
...
+    `,
+    'example.spec.ts': `
+      const { test } = pwt;
+
+      import * as components from './component';
+      import * as regular from './regular';
+
+      test('works', () => {
+        expect.soft(components.MyEnum.Value).toBe("value");
+        expect.soft(components.MyConstEnum.Value).toBe("value");
```
Comment (rwoll) [inline tests/playwright-test/babel.spec.ts:69]: @pavelfeldman What should we expect `components.MyConstEnum.Value` to be? It's currently "undefined", while those coming out of `regular.ts` (below) are defined as "value".

File: tests/playwright-test/babel.spec.ts [hunk 2]
```diff
@@ -36,3 +36,43 @@ test('should succeed', async ({ runInlineTest }) => {
   expect(result.passed).toBe(1);
   expect(result.failed).toBe(0);
 });
+
+test('should treat enums equally', async ({ runInlineTest }) => {
+  test.fail(true, 'https://github.com/microsoft/playwright/issues/13265');
```
Comment (mxschmitt) [inline tests/playwright-test/babel.spec.ts:41]: We started to do it differently: https://github.com/microsoft/playwright/blob/02cac8a0668c59078eb35029ed88c9f88e139428/tests/library/browsercontext-page-event.spec.ts#L189

File: tests/playwright-test/babel.spec.ts (hunk 1 above)
Comment (pavelfeldman) [reply tests/playwright-test/babel.spec.ts:69]: We don't compile TSX, we only stub those altogether for component testing. So this test documented present behavior.

**Review by pavelfeldman — APPROVED** [review]

File: tests/playwright-test/babel.spec.ts (hunk 2 above)
Comment (rwoll) [inline tests/playwright-test/babel.spec.ts:41]:
```suggestion
  test.info().annotations.push({ type: 'issue', description: 'https://github.com/microsoft/playwright/issues/13265' }); 
```

**Pull request merged**